api/external/compliance/reporting/reporting.proto

```diff
@@ -122,6 +122,28 @@ service ReportingService {
 		};
 	};
 
+	/*
+	Show Node Header Info From Report ID
+
+	Show specific details about node, report and metadate provided the report ID.
+	Supports filtering, but not pagination or sorting.
+
+	Authorization Action:
+	```
+	compliance:nodeheader:get
+	```
+	*/
+	rpc ReadNodeInfo(Query) returns (NodeReport) {
+		option (google.api.http) = {
+			post: "/api/v0/compliance/reporting/nodeheader/id/{id}"
```

Comment:
Since we are sending report Id in the URL and not passing any other information over the body, we can have GET API instead of POST.

Reply:
@Dmaddu We are sending a filter payload for filtering the end and start time. So we need POST api